### ISSUE TITLE
iterable<string, stirng> space parsing invalid

### ISSUE THREAD

**Issue by simPod** [slevomat/coding-standard#705]
```php
function a()
{
    /** @var iterable<string, string> $x */
    $x = b();

    return $x;
}
```

This reports $x as [useless variable](https://github.com/slevomat/coding-standard#slevomatcodingstandardvariablesuselessvariable-).

```php
function a()
{
    /** @var iterable<string,string> $x */
    $x = b();

    return $x;
}
```

this does not.

Shouldn't it also process the version with space after `,`?

**Comment by kukulich** [comment]
Duplicate of https://github.com/slevomat/coding-standard/issues/685
